Userland/Libraries/LibWeb/HTML/HTMLTableCellElement.idl

```diff
@@ -2,6 +2,8 @@
 
 interface HTMLTableCellElement : HTMLElement {
 
+    attribute unsigned long colSpan;
+    attribute unsigned long rowSpan;
```

Comment:
Your IDL here says unsigned long but the code above uses unsigned int. Is this an oversight, or does the Code Generator actually try to call with an unsigned int?

Reply:
The code generator turns unsigned long into u32. Might make sense to declare these methods as u32 since `unsigned int` is incorrect code style anyway.

Reply:
Gotcha. That sounds like suspicious behavior from the generator to me, might be worth a trip to the IDL spec outside the scope of this PR.

Reply:
Perhaps we could make the code generator spit out something like `IDL::UnsignedLong` and then we make sure it means what it should mean in a header somewhere. Anyways, that's :bikeshed: for somewhere else :)